androgee/androgee.py

```diff
@@ -20,8 +22,15 @@ async def spray(ctx, member: discord.Member = None):
     if member == None:
         await ctx.send(file=image)
     else:
-        message = f"{member.mention} was sprayed by {ctx.message.author.mention}"
-        await ctx.send(message, file=image)
+        if mod_role_id in [y.id for y in ctx.author.roles]:
+            await member.edit(mute=True)
+            message = f"{member.mention} was sprayed by {ctx.message.author.mention}"
+            await ctx.send(message, file=image)
+            await asyncio.sleep(10)
+            await member.edit(mute=False)
+        else:
+            message = f"{member.mention} was sprayed by {ctx.message.author.mention}"
```

Comment:
Let's change "sprayed" to "spritzered"

Reply:
I've set spritzered as a alias instead is that fine?

Reply:
nah `-spray` is fine still. But the word "sprayed" looks weird and "spritzered" sounds funnier

Reply:
oh okay